src/explore-education-statistics-admin/src/pages/release/content/components/EditableKeyStatDataBlock.tsx

```diff
@@ -76,7 +78,9 @@ export default function EditableKeyStatDataBlock({
     return (
       <EditableKeyStatDataBlockForm
         keyStat={keyStat}
-        keyStatisticGuidanceTitles={keyStatisticGuidanceTitles}
+        keyStatisticGuidanceTitles={keyStatisticGuidanceTitles?.filter(
+          keyStatTitle => keyStatTitle === keyStatisticId,
+        )}
```

Comment:
as before:

> I don't think this will be working currently, as we aren't setting `keyStatisticId` to the keyStat's title.  
> 
> We're wanting to remove the title of _this_ keyStat form the list of all keyStatisticGuidanceTitles, we can use `keyState.title` in the filter. eg:
> ```js
> keyStatisticGuidanceTitles?.filter(
>           keyStatTitle => keyStatTitle === keyStat.title)
> ```